openapi-validator.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -e
+for entry in ./public/doc/*.json
```

Comment:
Thanks @bzwei.  Is there any way this file could live in `.swagger-codegen`?

I'm not sure if that folder has special meaning and should only contains the `VERSION` file.  Thought it was worth asking so we don't end up collecting a bunch of files in the root folder.

Reply:
@gmcculloug Not sure I understand your recommendation. Do you want to move all openapi spec files under `.swagger-codegen`?
How about we further narrow down the match pattern to
```
for entry in ./public/doc/openapi-3-v*.json
```

Reply:
I can move the script to `.swagger-codegen`

Reply:
I was referring to moving this new script to that folder, to get it out of the root.
@mkanoor Do you see any issue with that?

Reply:
@gmcculloug @bzwei 
I think that .swagger-codegen is a left over from when we started using the server side generators, we are no longer using it. It might make more sense to put this in a separate directory that we can create .travis_scripts or .ci_scripts. I think we should delete .swagger-codegen directory from the approval-api side, it doesn't exist in catalog-api, we removed it a while back.

Reply:
moved to `.travis_scripts`